virttest/libvirt_xml/vm_xml.py

```diff
@@ -593,6 +594,13 @@ def get_device_class(type_name):
 
     def undefine(self, options=None, virsh_instance=base.virsh):
         """Undefine this VM with libvirt retaining XML in instance"""
+        # add --nvram in undefine for aarch64
```

Comment:
Because the arm guest boot up must have 
    <loader readonly='yes' type='pflash'>/usr/share/AAVMF/AAVMF_CODE.fd</loader>
    <nvram>/var/lib/libvirt/qemu/nvram/virt-tests-vm3_VARS.fd</nvram>
so undefine will need --nvram, it do not related to libvirt version, but the configure